fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -123,7 +122,8 @@ def solve_linprog(self, nu):
         dual_c = np.concatenate((b_ub, -b_eq))
         dual_A_ub = np.concatenate((-A_ub.transpose(), A_eq.transpose()), axis=1)
         dual_b_ub = c
-        dual_bounds = [(None, None) if i == n_constraints else (0, None) for i in range(n_constraints + 1)]  # noqa: E501
+        dual_bounds = [(None, None) if i == n_constraints else (0, None) for i in
```

Comment:
Was this split intentional?

Reply:
Oops, I think my IDE automatically did this.